goodtables/checks/missing_value.py

```diff
@@ -13,12 +13,16 @@
 
 @check('missing-value')
 def missing_value(cells):
+    """
+    missing-value: 	A row has less columns than the header.
+    """
     errors = []
 
     for cell in copy(cells):
 
         # Skip if cell has value
-        if cell.get('value') is not None:
+        # Skip if cell is missing value but belongs to blank header
+        if cell.get('value') or not cell.get('header'):
```

Comment:
I think it shouldn't be related. A header could be empty `''` in csv by mistake but it still creates a column

Reply:
@roll Thanks for reviewing

Missing value check provided the following definition in Readme.
 
<img width="815" alt="Screenshot 2019-10-24 at 10 35 34 PM" src="https://user-images.githubusercontent.com/24490498/67508477-bb61c500-f6ae-11e9-80e8-58233b4c50a4.png">

If csv had blank header, and if the empty cells under it are reported as missing values, it contradicted the definition of the missing values. 
Thus added the check. 

Please validate if empty cells under a blank header(which itself is not a proper header) can be reported as missing values.

If so, will make necessary changes and push.

Reply:
@SrividhyaGuru

> Please validate if empty cells under a blank header(which itself is not a proper header) can be reported as missing values.

Yes. This statement is correct. They have to. Sorry for the confusion We need to add a note a readme. The idea that the `headers` row determines a dimension of the table so, for example:

```csv
id,,question2
1,no,yes
2,yes,no
3,,no
```
This table has 3 columns and both the header and row 3 miss values for column 2

Reply:
@roll thanks for the clarification. Will update the code soon.

Reply:
@roll Have reverted the check for missing value check. Earlier missing value validator checked only for None and not for empty. Now the condition `if cell.get('value')` will cover both None and empty and report as missing values.
The build is breaking because of test_foreign_key.py.
This is due to the below code in **missing_value.py**

       # Add error
        error = Error('missing-value', cell)
        errors.append(error)

        # Remove cell
        cells.remove(cell)

The missing value check method after adding the missing value error, removes the cell itself. The rest of the validators dont get that cell at all. 
Thus foreign key check did not get a cell that had missing value for next_id

Is it necessary to remove the cell from all other validation if it is missing value?

Reply:
@roll Could you please confirm on the above so that we could make the necessary changes. If cells could be removed if it has missing values, it would impact the foreign key validator that plays after the missing value validator.

Reply:
Hi, thanks for your help.

I've been a little bit busy lately. I'm going to wrap my head around it this week or next week.

Reply:
Sure, thanks @roll